core-tests/src/test/java/org/mule/runtime/core/routing/MapSplitterTestCase.java

```diff
@@ -43,9 +45,13 @@ public void testSplit() throws Exception {
 
     mapSplitter.process(eventBuilder().message(of(testMap)).build());
 
-    assertEquals(3, splitPayloads.size());
-    assertTrue(splitPayloads.contains("one"));
-    assertTrue(splitPayloads.contains("two"));
-    assertTrue(splitPayloads.contains("three"));
+    assertThat(3, is(splitPayloads.size()));
```

Comment:
also, for now you may use `assertThat(splitPayloads.values(), hasSize(3)))`, that will show the contensts of the map on a failure